fruit.txt

```diff
@@ -5,4 +5,3 @@ guava
 pear
 pineapple
 strawberry
-tomato
```

Comment:
tomayto, tomahto

---
In reply to: [59634887](https://github.com/chwarr/playground/pull/3#discussion_r59634887) [](ancestors = 59634887)